cdap-app-fabric/src/main/java/io/cdap/cdap/app/store/ListSourceControlMetadataResponse.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2024 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.app.store;
+
+import io.cdap.cdap.proto.SourceControlMetadataRecord;
+import java.util.List;
+
+/**
+ * Represents a response containing a list of source control metadata records, next page token and
+ * last refresh time.
+ */
+
+public class ListSourceControlMetadataResponse {
+
+  private final List<SourceControlMetadataRecord> apps;
+  private final String nextPageToken;
+  private final Long lastRefreshTime;
```

Comment:
Its null if the refresh has not been run yet. I think it would be more appropiate to have default value as null in the API response than a actual long value

Reply:
I don't think it can ever be null. The value comes from `SourceControlMetadataRefresher.getLastRefreshTime`, which returns primitive long, and the implementation is defaulting it to `0L`